internal/statemachine/helper.go

```diff
@@ -922,22 +922,54 @@ func manualAddGroup(customizations []*imagedefinition.AddGroup, targetDir string
 // manualAddUser adds users in the chroot
 func manualAddUser(customizations []*imagedefinition.AddUser, targetDir string, debug bool) error {
 	for _, c := range customizations {
-		addUserCmd := execCommand("chroot", targetDir, "useradd", c.UserName)
 		debugStatement := fmt.Sprintf("Adding user \"%s\"\n", c.UserName)
+		var addUserCmds []*exec.Cmd
+
+		addUserCmd := execCommand("chroot", targetDir, "useradd", c.UserName)
```

Comment:
I know it might be a bit annoying for the users, but maybe we should print out a warning that we do not recommend this solution and that cloud-init is better and safer.

Reply:
I agree about warning users that using cloud-init for specific customization is the recommended way because they can do more powerful things. But in that case I am not really convinced our feature is "risky". I do not want to give users the feeling that this way of setting the user/password is unreliable. Because if it is then we should either not doing it or we should fix it I think.

Reply:
Agreed. Let's maybe not label it as risky, but maybe instead hint about using cloud-init instead in the yaml documentation?

Reply:
Good idea, especially since our documentation should now be more visible.